src/Registration.css

```diff
@@ -0,0 +1,113 @@
+.RegistrationForm {
+    padding: 1.50rem;
+    border-radius: 40px;
+    box-shadow: 1px 4px 12px rgba(0, 0, 0, 0.2);
+    background-color: #ffffff;
+    font-family: Arial, sans-serif;
+    display: flex;               
+    flex-direction: column;      
+    align-items: center;         
+    justify-content: center; 
+    position: relative;  
+    padding-left: 3rem;
+    padding-right: 3rem;
+    margin-top: 60px;
+  }
+
+  .RegistrationForm h1 {
+    font-size: 2.5rem;
+    text-align: center;
+    color: #35415D;
+    margin-bottom: 10px;
+    font-family: Inter, sans-serif;
+  }
+  
+  label {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    text-align: left;
+    margin-top: 20px;
+    margin-bottom: 5px;
+    color: #3a3a3a;
+  }
+  
+  .input-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start; /* Ensure items are left-aligned*/
+  }
+  
+  input {
+    width: 100%;
+    padding: 8px;
```

Comment:
have you tested with a really long string? Doesn't the text go behind the icon, if the icon is inside the field?